### PR TITLE
fix(ts-fixes): ts definition changes

### PR DESCRIPTION
**Pull request by priley86**
**What**:
A few ts definitions out of sync today... probably easiest to remove the `extends HTMLProps<HTMLDivElement>` piece for now. There is nested element types returned so not really necessary on these.

```
ERROR in /Users/priley/GitHub/web-ui/frontend/node_modules/@patternfly/react-core/dist/js/components/ContextSelector/ContextSelector.d.ts(4,18):
TS2430: Interface 'ContextSelectorProps' incorrectly extends interface 'HTMLProps<HTMLDivElement>'.
  Types of property 'onSelect' are incompatible.
    Type '(event: SyntheticEvent<Element>, value: ReactNode) => void' is not assignable to type '(event: SyntheticEvent<HTMLDivElement>) => void'.
ERROR in /Users/priley/GitHub/web-ui/frontend/node_modules/@patternfly/react-core/dist/js/components/ContextSelector/ContextSelectorItem.d.ts(3,18):
TS2430: Interface 'ContextSelectorItemProps' incorrectly extends interface 'HTMLProps<HTMLButtonElement>'.
  Types of property 'onClick' are incompatible.
    Type 'Function' is not assignable to type '(event: MouseEvent<HTMLButtonElement>) => void'.
      Type 'Function' provides no match for the signature '(event: MouseEvent<HTMLButtonElement>): void'.
ERROR in /Users/priley/GitHub/web-ui/frontend/node_modules/@patternfly/react-table/dist/js/components/Table/index.d.ts(18,53):
TS2305: Module '"/Users/priley/GitHub/web-ui/frontend/node_modules/@patternfly/react-table/dist/js/components/Table/utils/index"' has no exported member 'IExtra'.
```

**Additional issues**:

### CODE REVIEW

File: packages/patternfly-4/react-table/src/components/Table/utils/index.d.ts [hunk 1]
```diff
@@ -3,11 +3,12 @@ import { IRow, IExtra } from '../Table';
...
-  children: ReactNode
+  children: ReactNode;
 }
 
 export const sortable: (label: string, extra: IExtra) => ISortable;
 export const headerCol: () => { component: string };
 export const cellWidth: (width: string) => () => { className: string };
 export const expandable: (value: ReactNode, extra: IExtra) => ReactNode;
 export const isRowExpanded: (row: IRow, rows: Array<IRow>) => boolean | undefined;
+export { IExtra };
```
Comment (priley86) [inline packages/patternfly-4/react-table/src/components/Table/utils/index.d.ts]: @suomiy ok for now or did you have something else in mind?

Comment (suomiy) [reply packages/patternfly-4/react-table/src/components/Table/utils/index.d.ts]: why is this necessary? `IExtra` should be already exported from `Table.d.ts` and `index.d.ts`

Comment (priley86) [reply packages/patternfly-4/react-table/src/components/Table/utils/index.d.ts]:
```
ERROR in /Users/priley/GitHub/web-ui/frontend/node_modules/@patternfly/react-table/dist/js/components/Table/index.d.ts(18,53):
TS2305: Module '"/Users/priley/GitHub/web-ui/frontend/node_modules/@patternfly/react-table/dist/js/components/Table/utils/index"' has no exported member 'IExtra'.
```
can you try it? tested with :
```
    "@patternfly/patternfly": "1.0.240",
    "@patternfly/react-core": "2.9.0",
    "@patternfly/react-table": "1.3.2",
```

Comment (priley86) [reply packages/patternfly-4/react-table/src/components/Table/utils/index.d.ts]: ok, for temp fix has moved it to `index.d.ts` to match your inline-edit change for now:
https://github.com/patternfly/patternfly-react/pull/1227/files#diff-7d0e921725e74d478d1bee10116f6e8b

**Pull request merged**